web/frontend/src/lib/images/converter.js

```diff
@@ -0,0 +1,12 @@
+import { readFileSync, writeFileSync } from 'node:fs';
+
+const files = [
+  'base-marker.png',
+  'destination-marker.png',
+];
+
+for (const file of files) {
+  const buffer = readFileSync(file);
+  const base64String = buffer.toString('base64');
+  writeFileSync(file.replace('png', 'txt'), `data:image/png;base64,${base64String}`, 'utf8');
+}
```

Comment:
Fun with PNGs eh...someday we should use SVG?

Reply:
LOL

Reply:
(for others: I agree with Jason, SVGs can often be much simpler to use. In this case though since webgl is a a low level rasterization engine svgs end up requiring a lot more boilerplate and it's simpler to use bitmaps as textures)